src/Symfony/Component/Messenger/Middleware/HandleMessageInNewTransactionMiddleware.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Middleware;
+
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Exception\QueuedMessageHandlingException;
+use Symfony\Component\Messenger\Stamp\Transaction;
+
+/**
+ * Allow to configure messages to be handled in a new transaction.
+ * I.e, messages dispatched from a handler with a Transaction stamp will actually be handled
```

Comment:
missing blank line to split title from description

Reply:
Thank you